open-hackathon-server/src/hackathon/user/user_manager.py

```diff
@@ -219,9 +248,14 @@ def __validate_token(self, token):
         :rtype: User
         :return user related to the token or None if token is invalid
         """
-        t = self.db.find_first_object_by(UserToken, token=token)
-        if t is not None and t.expire_date >= self.util.get_now():
-            return t.user
+        if "authenticateUser" in g and g.authenticateUser:
```

Comment:
Please rename `authenticateUser` to `authenticated`. `authenticateUser` sounds like an instance of `User`